test/queries_generator/aws_queries.txt

```diff
@@ -0,0 +1,10 @@
+select lower(lower('  %%AbCdEfGhIjKlMnOpQrStUvWxYz##  ')) from s3object;
```

Comment:
some queries containing only constant values which means their results is static (no change between rows)
those kind of queries not need an input object (actually any input object will produce the same result)